sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2775,6 +2775,16 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val LEGACY_CENTRAL_MOMENT_AGG_BEHAVIOR =
+    buildConf("spark.sql.legacy.centralMomentAgg.enabled")
```

Comment:
Could you update the migration guide, too?

Reply:
Looks we don't need the suffix `.enabled` for following the other legacy configs.

Reply:
Also, could you move this config close to the other legacy configs?